shells/tests/artifacts/Common/List.manifest

```diff
@@ -0,0 +1,66 @@
+// @license
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+interface HostedParticleInterface
```

Comment:
do we now have 2 sets of these manifests - both under `runtime/tests/artifacts/Common/` and here `shells/tests/artifacts/Common/`? or can we remove them from `runtime/tests` maybe?

Reply:
At first blush, it looks like the dependencies are not that simple. I hoped to work though this in a separate PR.